platform/jn516x/dev/micromac-radio.c

```diff
@@ -57,14 +57,24 @@
 #include "JPT.h"
 #include "PeripheralRegs.h"
 
-void vMMAC_SetChannelAndPower(uint8 u8Channel, int8 i8power);
-
```

Comment:
Actually we want as the signature of the function has also changed :)

In the current SDK it is :

    PUBLIC void vMMAC_SetChannelAndPower(uint8 u8Channel, int i8TxPower);

So, int instead of int8 for TxPower.